ebos/eclactionhandler.hh

```diff
@@ -62,6 +63,10 @@ public:
                     double sim_time,
                     const TransFunc& updateTrans);
 
+  //! \brief Evaluates UDQ statements.
+  void evalUDQ(const unsigned episodeIdx,
+               UDQState& udq_state);
+
```

Comment:
Can you name this `evalUDQAssignments()` or something along those lines please?  Unless we alter the implementation, the function only runs UDQ "assign" statements and does not touch UDQ "define" statements and that's an important distinction.

Reply:
done.